std/container/slist.d

```diff
@@ -354,15 +354,18 @@ Returns: The number of elements inserted
 
 Complexity: $(BIGOH m), where $(D m) is the length of $(D stuff)
      */
-    size_t insertFront(Stuff)(Stuff stuff)
+    size_t insertFront(Stuff)(scope Stuff stuff)
     if (isInputRange!Stuff && isImplicitlyConvertible!(ElementType!Stuff, T))
     {
         initialize();
         size_t result;
         Node * n, newRoot;
         foreach (item; stuff)
         {
-            auto newNode = new Node(null, item);
+            static if (isSomeString!T)
+                auto newNode = new Node(null, item.dup);
```

Comment:
Didn't we agree that this is bad because we duplicate memory without the user explicitly agreeing too.
Why not `version(DIP1000) else` out the test for now?  And/or report the DMD error I reduced here: https://github.com/dlang/phobos/pull/6295#issuecomment-375813328